flake.nix

```diff
@@ -225,8 +224,7 @@
             cargoExtraArgs = "${cargoBuildExtraArgs} --workspace";
             # pyo3 needs a Python interpreter in the build environment
             # https://pyo3.rs/v0.17.3/building_and_distribution#configuring-the-python-version
-            # XXX: I still need to add "-isystem ${pkgs.nix}/include/nix" to $NIX_CFLAGS_COMPILE
-            buildInputs = [ pkgs.python3 pkgs.nixVersions.nix_2_16 pkgs.boost ];
+            buildInputs = with pkgs; [ python3 pkg-config nixVersions.nix_2_16 boost ];
```

Comment:
Cross-building is probably broken for other reasons already, but `pkg-config` should be in `nativeBuildInputs`. That might be true for `python3` as well, or that might need to be in both `buildInputs` and `nativeBuildInputs` for `pyo3`.